wp-content/themes/citylimits/tribe-events/~single-event.php

```diff
@@ -0,0 +1,88 @@
+<?php
```

Comment:
What purpose does this `~single-event.php` file serve? It appears to be a duplicate of `single-event.php`.